Sources/protoc-gen-swift/ReservedWords.swift

```diff
@@ -20,15 +20,14 @@ import PluginLibrary
 private let reservedTypeNames: Set<String> = {
     () -> Set<String> in
     var names: Set<String> = [
-        "anyTypeURL",
+        "SwiftProtobuf",
```

Comment:
Maybe a general comment on where this come from and why?  i.e. - call out the functions that have to block sub types, etc.